src/VirtoCommerce.Platform.Security/Repositories/SecurityDbContext.cs

```diff
@@ -37,5 +40,24 @@ protected override void OnModelCreating(ModelBuilder builder)
             builder.Entity<IdentityRoleClaim<string>>().Property(x => x.RoleId).HasMaxLength(128);
             builder.Entity<IdentityUserToken<string>>().Property(x => x.UserId).HasMaxLength(128);
         }
+
+        #region override Save*** methods to catch save events in Triggers, otherwise ApplicationUser not be catched because SecurityDbContext can't inherit DbContextWithTriggers
+        public override int SaveChanges()
+        {
+            return this.SaveChangesWithTriggers(base.SaveChanges, acceptAllChangesOnSuccess: true);
```

Comment:
Remove `this` keywords

Reply:
Please explain.
It's extension method, therefore is no way to remove `this`.

Reply:
OK